src/commands/app/logs.js

```diff
@@ -20,57 +19,37 @@ const BaseCommand = require('../../BaseCommand')
 class Logs extends BaseCommand {
   async run () {
     const { flags } = this.parse(Logs)
-    this.foundLogs = false
-    this.aioConfig = aioConfig.get() || {}
-    this.validateConfig()
-    await this.getLogs(flags.limit)
-    if (this.foundLogs) { this.log('✔ Finished fetching logs!') } else { this.log('No Logs Found') }
-  }
 
-  async getLogs (limit) {
-    const options = {
-      apihost: this.aioConfig.runtime.apihost,
-      api_key: this.aioConfig.runtime.auth,
-      namespace: this.aioConfig.runtime.namespace
-    }
-    const ow = openwhisk(options)
-    // get activations
-    const listOptions = { limit: limit, skip: 0 }
-    const activations = await ow.activations.list(listOptions)
-    for (let i = 0; i < activations.length; i++) {
-      await this.getActivationLogs(activations[i], ow)
-    }
-  }
+    const scripts = AppScripts({ listeners: {} })
 
-  async getActivationLogs (activation, ow) {
-    const logOptions = { activationId: activation.activationId }
-    const results = await ow.activations.logs(logOptions)
-    // send fetched logs to console
-    if (results.logs && results.logs.length > 0) {
-      this.foundLogs = true
-      const logger = this.log
-      logger(activation.name + ':' + activation.activationId)
-      results.logs.forEach(function (log) {
-        logger(log)
-      })
+    const logOptions = {
+      logger: this.log,
+      limit: flags.limit
     }
-  }
 
-  validateConfig () {
-    if (!this.aioConfig) { throw new Error('Missing aio config') } else if (!this.aioConfig.runtime || !this.aioConfig.runtime.auth || !this.aioConfig.runtime.namespace) { throw new Error('Missing aio runtime config') }
+    try {
+      const foundLogs = await scripts.logs([], logOptions)
+      if (foundLogs) {
+        this.log('✔ Finished fetching logs!')
```

Comment:
why do we keep adding extra text? what value does this add?

Reply:
I just reproduce the previous behavior of the log command, didn't want to change anything behavior wise.
but agree there is not much value, will remove it.